draft-packages/modal/KaizenDraft/Modal/Primitives/GenericModalSection.tsx

```diff
@@ -1,17 +1,29 @@
 import * as React from "react"
+import classNames from "classnames"
 
 import styles from "./GenericModalSection.scss"
 
 export interface GenericModalSectionProps {
   readonly unpadded?: boolean
+  readonly inputEdit?: boolean
   readonly children: React.ReactNode
 }
 
 type GenericModalSection = React.FunctionComponent<GenericModalSectionProps>
 
 const GenericModalSection: GenericModalSection = ({
   unpadded = false,
+  inputEdit = false,
   children,
-}) => <div className={unpadded ? undefined : styles.padded}>{children}</div>
+}) => (
+  <div
+    className={classNames(
+      unpadded ? undefined : styles.padded,
+      inputEdit && styles.inputEdit
```

Comment:
These 2 lines read weird, because they do the same thing in 2 ways. For consistency and readability, I'd suggest always using the object syntax `classNames` offers - and which is used e.g. on line 140 in `draft-packages/modal/KaizenDraft/Modal/Presets/ConfirmationModal.tsx `, which would make this:

```
className={classNames({
  [styles.padded]: !unpadded,
  [styles.inputEdit]: inputEdit
})
```
This is nicer because it's always `[class]: condition`.